src/svgSprite.ts

```diff
@@ -907,5 +907,31 @@ export const svgSprite = (): string => `
     <rect width="72" height="4" rx="2" fill="#DEE3EB"/>
     <rect width="41" height="4" rx="2" fill="#42474E"/>
   </symbol>
+  <symbol viewBox="0 0 12 16" id="svg-md-editor-mark">
```

Comment:
Can you add empty lines between symbols, please?
And please import icons 24x24, you can find them [here](https://www.figma.com/file/LXl9BAJSwtgWuzkxqOqvX8/ML1-Design-System?node-id=53015-36793&t=AR0FKTZEKXAlglih-0)

Reply:
done